release/build_release.sh

```diff
@@ -69,6 +69,7 @@ function main {
 
 	lc_background_run decrement_module_versions
 	lc_background_run update_release_tool_repository
+	lc_background_run create_release_properties_file
```

Comment:
IMO this should run after the repo is updated. Otherwise, there is no guarantee that it works with the right data. Correct me if I am wrong.